float32 speed
string suffix
bool always
---
int8 Result